.github/workflows/coverage.yml

```diff
@@ -19,22 +19,35 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+
       - uses: actions-rs/toolchain@v1
         with:
           profile: minimal
           toolchain: nightly
           override: true
           components: llvm-tools-preview
+
       - uses: actions-rs/install@v0.1
         with:
           crate: cargo-binutils
           version: 0.3.3
+
       - uses: actions-rs/install@v0.1
         with:
           crate: rustfilt
           version: 0.2.1
+
+      - name: Install dashboard frontend
+        run: |
+          git submodule init
+          git submodule update --recursive --remote
+          cd bee-node/src/plugins/dashboard/frontend/
+          npm install
+          npm run build-bee
```

Comment:
Is a `cd -` missing here?

Reply:
Tbh I initially forgot it but since it's working fine on all other jobs I believe each `run` is done in a subshell and it doesn't seem necessary to do it

Reply:
Yeah, each command that follows `run` starts in the working directory of the environment.

Reply:
Ah, perfect. Thanks for clearing that up.